Modules/Module_Import/Import/xASL_imp_DetermineStructureFromSourcedata.m

```diff
@@ -437,28 +443,28 @@ function xASL_imp_PrintRun(x,thisSubject,thisVisit,thisRun)
 	else
 		x = xASL_imp_AddSessionIDListAndScansOfRun(x, sFieldName, vFieldName, vSessionName, thisSession);
     end
-
-
 end
 
+% -----------------------------------------------------------------
+%% Add session ID List and scans of run
+% -----------------------------------------------------------------
 function x = xASL_imp_AddSessionIDListAndScansOfRun(x, sFieldName, vFieldName, vSessionName, thisRegExp)
-% Iterate over scan IDs
-x.importOverview.(sFieldName).(vFieldName).(vSessionName).scanIDs = {};
-scanSessionTable = x.importOverview.(sFieldName).(vFieldName).scanSessionTable;
-iSessionNew = 1;
-for iID=1:size(scanSessionTable,1)
-	thisToken = scanSessionTable{iID,1};
-	thisScanID = scanSessionTable{iID,2};
-	% Iterate over the sessions of this run
-	for iSession=1:numel(x.importOverview.(sFieldName).(vFieldName).sessions)
-		currentID = x.importOverview.(sFieldName).(vFieldName).sessions{iSession};
-		if ~isempty(regexpi(currentID, thisRegExp)) && strcmp(currentID, thisToken)
-			x.importOverview.(sFieldName).(vFieldName).(vSessionName).scanIDs{iSessionNew} = thisScanID;
-			x.importOverview.(sFieldName).(vFieldName).(vSessionName).ids{iSessionNew} = currentID;
-			iSessionNew = iSessionNew + 1;
-			continue;
+    % Iterate over scan IDs
```

Comment:
This is annoying because I cannot see if you changed the content or just added a tab. I know that many people like an extra indent after function but I think it is just a waste of horizontal space (and extra useless commits in this case :)

Reply:
Not me. I hate extra indent for a new function and I'm usually removing them :) Not sure why this happened - probably a mistake... Cause nothing changed here